Makefile

```diff
@@ -332,6 +332,26 @@ else
 endif
 	docker tag antrea/antrea-ubuntu:$(DOCKER_IMG_VERSION) antrea/antrea-ubuntu
 
+.PHONY: build-ubuntu-controller
+build-ubuntu-controller:
+	@echo "===> Building antrea/antrea-ubuntu Docker image <==="
```

Comment:
Echo message should be include controller as this step is building image for controller